CryptoWallet/Extensions/UIViewController+.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 
 extension UIViewController {
     func showAlertView(title: String?, message: String?, cancelButton: String?, otherButtons: [String]? = nil, type: UIAlertController.Style = .alert, cancelAction: (() -> Void)? = nil, otherAction: ((Int) -> Void)? = nil) {
-        let alertViewController = UIAlertController(title: title ?? Constants.appName.rawValue ,
+        let alertViewController = UIAlertController(title: title ?? Constants.appName ,
```

Comment:
gán giá trị mặc định cho title như này:
`func showAlertView(title: String = Constants.appName, message: String?, cancelButton: String?, otherButtons:` 
```suggestion
        let alertViewController = UIAlertController(title: title,
```